c/meterpreter/source/extensions/stdapi/server/sys/registry/registry.c

```diff
@@ -463,6 +463,7 @@ static void set_value(Remote *remote, Packet *packet, HKEY hkey)
 		switch (valueType) {
 			case REG_SZ:
 			case REG_EXPAND_SZ:
+			case REG_MULTI_SZ:

```

Comment:
This works fine for the single-string case of MULTI_SZ and is definitely more correct than what we have now. If you want to handle 2 or more strings, this needs some extra work. Could you just add a comment noting that this might not work for more than one string? Thanks!